src/main/java/soot/SootMethodRef.java

```diff
@@ -55,26 +55,22 @@
    */
   public @Deprecated Type returnType();
 
-  public boolean isStatic();
```

Comment:
was this removal intentionally?

Reply:
Yes, because it's redundant. The inherited interface ```SootMethodInterface``` already defines this method.